### PR TITLE
radiobutton

### PR DESCRIPTION
**Pull request by Amitrawat21**
radiobutton of gender selection and radio button of money selection

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-import './components/buttons/index' // import all buttons component here to the entry point for the npm package
\ No newline at end of file
+import './components/buttons/index' // import all buttons component here to the entry point for the npm package
```
Comment (rohan-arya-w) [inline src/index.ts]: remove this line @Amitrawat21

File: src/interface/RadioButton/GenderSelectionInterfaceTwo.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ import { Option } from "./GenderSlectionInterface";
```
Comment (rohan-arya-w) [inline src/interface/RadioButton/GenderSelectionInterfaceTwo.ts]: Why do we need gender selection interface

File: src/components/RadioButton/RadioButtonComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import { responsiveHeight , responsiveWidth } from 'react-native-responsive-dimensions';
+import { RadioButtonComponentProps } from '../../interface/RadioButton/RadioButtonInterface';
+
+export const CustomRadioButtonComponent: React.FC<RadioButtonComponentProps> = ({
+  options,
+  checkedColor,
+  uncheckedColor,
+  containerStyle = {},
+  fontWeight,
+  textfontSize,
```
Comment (tabassum2507) [inline src/components/RadioButton/RadioButtonComponent.tsx]: camelcase

File: src/components/RadioButton/RadioButtonComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  title,
+  textPadding,
+  direction
```
Comment (tabassum2507) [inline src/components/RadioButton/RadioButtonComponent.tsx]: direction? give proper name

File: src/components/RadioButton/RadioButtonComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    )}
+  
+    <View style={{ ...optionContainerStyle }}>
+      {options?.map((option) => (
+        <View
+          key={option.value}
+          style={{
+            alignItems: 'center',
+            flexDirection: 'row',
+            marginVertical: 5,
```
Comment (tabassum2507) [inline src/components/RadioButton/RadioButtonComponent.tsx]: take the marginVertical in responsive width

File: src/interface/RadioButton/RadioButtonInterface.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (tabassum2507) [inline src/interface/RadioButton/RadioButtonInterface.ts:13]: align this section and also remove extra space

**Review by tabassum2507 — CHANGES_REQUESTED** [review]

File: src/interface/RadioButton/RadioButtonInterface.ts (hunk 6 above)
Comment (Amitrawat21) [reply src/interface/RadioButton/RadioButtonInterface.ts:13]: resolved

File: src/components/RadioButton/RadioButtonComponent.tsx (hunk 3 above)
Comment (Amitrawat21) [reply src/components/RadioButton/RadioButtonComponent.tsx]: done

File: src/components/RadioButton/RadioButtonComponent.tsx (hunk 5 above)
Comment (Amitrawat21) [reply src/components/RadioButton/RadioButtonComponent.tsx]: resolved

File: src/components/RadioButton/RadioButtonComponent.tsx (hunk 4 above)
Comment (Amitrawat21) [reply src/components/RadioButton/RadioButtonComponent.tsx]: resolved

**Review by tabassum2507 — APPROVED** [review]

**Review by rohan-arya-w — APPROVED** [review]

**Pull request merged**